exercises/01.managing-ui-state/README.mdx

```diff
@@ -10,7 +10,7 @@ how that works with React:
 
 ![render --> set up the DOM --> user interacts --> state changes --> re-render --> update the DOM --> return to user interacts](/react-app-lifecycle.png)
 
-From there's a cycle of user interaction, state changes, and re-rendering. This
+You can see from this diagram that there's a cycle of user interaction, state changes, and re-rendering. This
```

Comment:
sentence seemed incomplete

Reply:
Good point. I'll simplify it a bit further. Thanks!